cogs/vip.py

```diff
@@ -18,6 +18,46 @@
 class VIP(Cog):
     cog_command_error = cog_error_handler
 
+    #check if vip is still valid
+	#this is probably going to need to be threaded?
+    @Cog.listener()
+    async def on_message(self, message):
+
+        #vip role
+        vip_role = discord.utils.get(message.guild.roles, name="VIP")
+
+        #stop bot from replying to itself :)
+        if message.author.bot:
+            return
+		
+        #don't bother checking if they are still a vip if they don't have the role in the first place...
+        if vip_role in message.author.roles:
+
+            #connect to xf database
+            database = databases.Database(global_config.databases.tf2maps_site)
```

Comment:
We prob dont want to be spinning up new database connections on every discord message. I could see that leading very quickly to socket exhaustion